docs/user/agent/iscsi-boot.md

```diff
@@ -0,0 +1,84 @@
+# iSCSI Boot
+
+### What is iSCSI?
+iSCSI (Internet Small Computer Systems Interface) is a network storage protocol that allows clients (initiators) to send SCSI commands to storage devices (targets) over IP networks. iSCSI provides a centralized and cost-effective storage management to use remote storage as if it were local disk storage.
+
+
+### Supported since version: OpenShift 4.18
+
+## Note: 
+- Static networking is currently not supported when using iSCSI. This is because coreos-installer does not support the `--copy-network` when booting off a separate iSCSI network, see https://github.com/coreos/coreos-installer/issues/1389. 
+- ISCSI currently only supported on x86 platforms. See https://issues.redhat.com/browse/MGMT-19307 
+
+## Key Systemd Services for iSCSI Boot
+
+### `iscsistart.service`
+- **Path**: [`data/data/agent/systemd/units/iscsistart.service`](https://github.com/openshift/installer/blob/main/data/data/agent/systemd/units/iscsistart.service)
+- **Purpose**: Initiates the iSCSI boot process by preparing and connecting to the iSCSI target. Calls the iscsistart utility which allows an iSCSI session to be started for booting off an iSCSI target
+- **Platform**: non OCI platforms such as baremetal, none
+
+### `iscsiadm.service`
+- **Path**: [`data/data/agent/systemd/units/iscsiadm.service`](https://github.com/openshift/installer/blob/main/data/data/agent/systemd/units/iscsiadm.service)
+- **Purpose**: Calls the iscsiadm utility that will start iscsid to manage the iSCSI session needed to install the final rhcos image
+- **Platform**: non OCI platforms such as baremetal, none
+
+You can refer to the agent installer services for an example of the installation workflow, particularly for non-OCI platforms, here. https://github.com/openshift/installer/blob/main/docs/user/agent/agent_installer_services-install_workflow.png
+
+### `oci-eval-user-data.service`
+- **Path**: [`data/data/agent/systemd/units/oci-eval-user-data.service`](https://github.com/openshift/installer/blob/main/data/data/agent/systemd/units/oci-eval-user-data.service)
+- **Purpose**: Executes a script passed via OCI cloud-init user data. This is **required only for OCI**. The script is maintained by OCI and is available here [`terraform-stacks/shared_modules/compute/userdata
+/iscsi-oci-configure-secondary-nic.sh`](https://github.com/oracle-quickstart/oci-openshift/blob/main/terraform-stacks/shared_modules/compute/userdata/iscsi-oci-configure-secondary-nic.sh)
+- **Platform**: OCI
+- **Note**: Without this service, network configuration will fail, causing the cluster installations to hang.
+
+
+## iSCSI Boot on External OCI
+
+To test and configure iSCSI boot on OCI:
```

Comment:
I think screenshots of each of the steps below or a video would be helpful. For example, it is not clear to me how to ensure the rendezvous IP is configured on the secondary NIC. And how does one include the oracle maintained script in step 5.